src/outfit/barf.ts

```diff
@@ -0,0 +1,153 @@
+import { Outfit, OutfitSpec } from "grimoire-kolmafia";
+import {
+  cliExecute,
+  inebrietyLimit,
+  Item,
+  myClass,
+  myFury,
+  myInebriety,
+  retrieveItem,
+  toJson,
+  totalTurnsPlayed,
+} from "kolmafia";
+import {
+  $class,
+  $familiar,
+  $item,
+  $items,
+  $skill,
+  get,
+  getKramcoWandererChance,
+  have,
+} from "libram";
+import { barfFamiliar } from "../familiar";
+import { chooseBjorn } from "./bjorn";
+import { bonusGear } from "./dropsgear";
+import {
+  bestBjornalike,
+  BonusEquipMode,
+  cleaverCheck,
+  validateGarbageFoldable,
+  valueOfItem,
+  valueOfMeat,
+} from "./lib";
+
+function chooseGun({ familiar }: Outfit) {
+  if (familiar === $familiar`Robortender` && have($item`love`)) return $item`love`;
+  if (!have($item`ice nine`)) {
+    cliExecute("refresh inventory");
+    retrieveItem($item`ice nine`);
+  }
+
+  return have($item`ice nine`) ? $item`ice nine` : null;
+}
+
+function gunSpec(outfit: Outfit) {
+  if (!outfit.canEquip($item`unwrapped knock-off retro superhero cape`)) {
+    return { available: false, items: [] };
+  }
+
+  const gun = chooseGun(outfit);
+  if (!gun) return { available: false, items: [] };
+
+  return {
+    available: true,
+    items: {
+      back: $item`unwrapped knock-off retro superhero cape`,
+      equip: [gun],
```

Comment:
The wiki says that the gun needs to be in the mainhand. But I see the old garbo code also didn't specify this so maybe that is incorrect.

Reply:
I think it's mostly that we are exceedingly unlikely to dual wield ranged weapons. but I'll make it a weapon